pkg/controller/sync/controller.go

```diff
@@ -700,15 +714,20 @@ func clusterVersions(newVersions map[string]string, oldVersions []fedv1a1.Cluste
 	return versions
 }
 
-type clusterObjectAccessorFunc func(clusterName string) (interface{}, bool, error)
+// clusterOperations returns the list of operations needed to synchronize the
+// state of the given object to the provided clusters.
+func (s *FederationSyncController) clusterOperations(selectedClusters, unselectedClusters []string,
+	template, override pkgruntime.Object, key string,
+	clusterVersions map[string]string) ([]util.FederatedOperation, error) {
 
-// clusterOperations returns the list of operations needed to synchronize the state of the given object to the provided clusters
-func clusterOperations(adapter federatedtypes.FederatedTypeAdapter, selectedClusters, unselectedClusters []string, template, override pkgruntime.Object, key string, clusterVersions map[string]string, accessor clusterObjectAccessorFunc) ([]util.FederatedOperation, error) {
 	operations := make([]util.FederatedOperation, 0)
+	accessor := func(clusterName string) (interface{}, bool, error) {
```

Comment:
I don't think accessor is necessary where it's not passed in - just use the function directly.

Reply:
It's definitely not necessary. I just left this one only to save some line width space since each invocation is quite lengthy.